mmv1/templates/terraform/examples/identity_platform_tenant_monitoring.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_identity_platform_tenant" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Looks like the error is with this resource, the project is being defaulted to the project that all the CI tests run in rather than a new one which causes it to fail with:

=== CONT  TestAccIdentityPlatformTenant_identityPlatformTenantMonitoringExample
    provider_test.go:315: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error creating Tenant: googleapi: Error 400: INVALID_PROJECT_ID
        
          with google_identity_platform_tenant.tenant,
          on terraform_plugin_test.tf line 3, in resource "google_identity_platform_tenant" "tenant":
           3: resource "google_identity_platform_tenant" "tenant" {

Reply:
Hmm right, what do you suggest I do in this case?

Reply:
That error happens because in the response the monitoring block isn't included for some reason. Do you know why this would be?
```
{
  "name": "projects/my-project/tenants/tenant-25ocl",
  "displayName": "tenant",
  "allowPasswordSignup": true,
  "inheritance": {}
}
```

Reply:
That's very weird, the response should contain the monitoring block, https://cloud.google.com/identity-platform/docs/reference/rest/v2/projects.tenants/patch#response-body